Shortcuts/apps.json

```diff
@@ -991,6 +991,24 @@
         "appDetectionInfo": {
           "applicationIdentifier": "\\\\Todoist\\.exe$"
         }
+      },
+      {
+        "internalName": "Files",
+        "appDisplayName": "Files",
+        "logo16": "ms-appx:///Assets/Shortcuts/Icons/Square44x44Logo.targetsize-16.png",
```

Comment:
Can we please rename these files to something like `Files-16.png` and `Files-64.png` ? I will try to change CtrlHelp Editor so it does it automatically in the future.

Reply:
Done 👍